microprofile/cdi/src/main/java/module-info.java

```diff
@@ -23,6 +23,8 @@
     // needed for Unsafe used from Weld
     requires jdk.unsupported;
     requires java.logging;
+    // weld requires java.sql.Data and we fail if not on classpath
```

Comment:
Hmm; no such class. Do you mean `java.sql.Date`?

Reply:
yes